dispatcher/backend/src/routes/errors/__init__.py

```diff
@@ -92,6 +92,13 @@ def handler(e):
 
 # 500
 class InternalError(Exception):
+    def __init__(self, message: str = None):
+        self.message = message
+
     @staticmethod
     def handler(e):
+        if isinstance(e, BadRequest) and e.message is not None:
```

Comment:
Yes this is a typo, should be `InternalError`.
This is used in src/routes/workers/worker.py when "something bad happened, the worker has been set but can't be found", i.e. when the return code must be 500 but we have something to say about what bad happened

Reply:
And I consider that an InternalError is more appropriate than an BadRequest in such a case because the other system is not responsible of the error, this is just that something is bad in our code, so it should be a 5xx